src/string_map.c

```diff
@@ -111,8 +111,8 @@ rcutils_string_map_get_capacity(const rcutils_string_map_t * string_map, size_t
     string_map->impl, "invalid string map",
     return RCUTILS_RET_STRING_MAP_INVALID, rcutils_get_default_allocator())
   RCUTILS_CHECK_ARGUMENT_FOR_NULL(
-    capacity, RCUTILS_RET_INVALID_ARGUMENT, rcutils_get_default_allocator())
-  *capacity = string_map->impl->capacity;
+    capacity, RCUTILS_RET_INVALID_ARGUMENT, rcutils_get_default_allocator()) *
+  capacity = string_map->impl->capacity;
```

Comment:
This is really hard to read. Can you work around this by doing something like this instead?

```
{
  *capacity = string_map->impl->capacity;
}
```

or 

```
// comment to keep style checker sane
*capacity = string_map->impl->capacity;
```